server/controllers/followersController.js

```diff
@@ -156,4 +167,16 @@ async function updateFollowersOfUser(newFollowers, unfollowers, user) {
     })
     console.log("Cleaning unfollower refs")
     await User.updateOne({ _id: user._id }, { $pull: { followers: { $in: unfollowerIds } } });
+}
+
+/**
+ * Function to get followers
+ * 
+ * @returns {Promise} resolving to the followers of user.
+ * @param {String} userId
+ */
+async function getFollowers(userId){
+    const accessToken = await getAccessToken();
+    const currentFollowers = await requestFollowers(accessToken, userId);
+    return currentFollowers;
 }
```

Comment:
Might be used in **getFollowersUnfollowers** controller function.

Reply:
yepp